src/components/Tooltip/Tooltip.tsx

```diff
@@ -43,8 +52,91 @@ export const Tooltip = ({
     addIndicator = false,
     markdownEnabler = "\n\n",
     markdownProps,
-    ...otherProps
+    usePlaceholder,
+    hoverOpenDelay = 500,
+    ...otherTooltipProps
 }: TooltipProps) => {
+    const placeholderRef = React.useRef(null);
+    const eventmemory = React.useRef<null | "afterhover" | "afterfocus">(null);
+    const searchId = React.useRef<null | string>(null);
+    const swapdelayTime = 100;
+    const [placeholder, setPlaceholder] = React.useState<boolean>(
+        otherTooltipProps.disabled !== true &&
+            otherTooltipProps.defaultIsOpen !== true &&
+            otherTooltipProps.isOpen !== true &&
+            typeof otherTooltipProps.renderTarget === "undefined" &&
+            hoverOpenDelay > swapdelayTime &&
+            (usePlaceholder === true || (typeof content === "string" && usePlaceholder !== false))
+    );
+
+    const targetClassName =
+        `${eccgui}-tooltip__wrapper` +
+        (className ? " " + className : "") +
+        (addIndicator === true ? " " + BlueprintClasses.TOOLTIP_INDICATOR : "");
+
+    React.useEffect(() => {
+        if (placeholderRef.current !== null) {
+            const swap = (ev: MouseEvent | globalThis.FocusEvent) => {
+                const swapdelay = setTimeout(() => {
+                    // we delay the swap to prevent unwanted effects
+                    // (e.g. forced mouseover after the swap but the curser is already somewhere else)
+                    eventmemory.current = ev.type === "focusin" ? "afterfocus" : "afterhover";
+                    searchId.current = Date.now().toString(16) + Math.random().toString(16).slice(2);
+                    setPlaceholder(false);
+                }, swapdelayTime);
+                if (placeholderRef.current !== null)
+                    (placeholderRef.current as HTMLElement).addEventListener(
+                        ev.type === "focusin" ? "focusout" : "mouseleave",
+                        () => clearTimeout(swapdelay)
+                    );
+            };
+            (placeholderRef.current as HTMLElement).addEventListener("mouseenter", swap);
+            (placeholderRef.current as HTMLElement).addEventListener("focusin", swap);
+        }
+    }, [!!placeholderRef.current]);
+
+    const refocus = React.useCallback((node) => {
```

Comment:
Is it possible that the `refocus` function gets called even though the element is not focused anymore, i.e. the focus event above triggers and `eventMemory` is set to `focusin`. Then the element loses focus, but `refocus` is called because the ref changes on another way?
Maybe it makes sense to set the `eventMemory` to `undefined` in the `focusout` event listener.